### ISSUE TITLE
[Accessibility] The <html> element does not have a lang attribute

### ISSUE THREAD

**Issue by plabanTest1** [gonfunko/scratch-blocks#145]
Accessibility tools used: Lighthouse & axe DevTools

Ensure the HTML element has a lang attribute

**Comment by gonfunko** [comment]
The page we use for testing is just a bootstrap/test environment, and is not the actual page that Scratch serves, which does include this.
